db/seeds.rb

```diff
@@ -116,7 +116,7 @@
                         brand: 'K2',
                         image: File.open('app/assets/images/www_144.png')
                       },
-                      { name: 'FASTPLANT SNOWBOARD',
+                      { name: 'FASTPLANT',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.